clients/java/src/main/java/io/camunda/zeebe/client/api/search/SearchRequestPage.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.search;
+
+import java.util.List;
+
+public interface SearchRequestPage {
+
+  /** Start the page from. */
+  SearchRequestPage from(final Integer value);
```

Comment:
❓ We don't have `from` defined in the REST API guidelines so far. What is this used for?

Reply:
It can be used for a classic pagination without search after, however, I can remove it if not needed.

Reply:
Understood. Do we offer this on the controller/server side?

Reply:
Yes, it offers it already.

Reply:
Ah OK, I didn't know 👍 Let's keep it then and add it to the REST guidelines 🙂

Reply:
I didn't add it to the guidelines yet because we weren't sure how complex it would be to add it to the search controllers.

Reply:
@romansmirnov, would you mind adding this to the REST guidelines by creating a proposal, describing how to define and use this in search and response? 🙂 I can take care of documenting this in the guidelines then 👍